lang/de/lang.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * German language file for findologicxmlexport plugin
+ *
+ * If any bugs occur, please submit a new issue
+ * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
+ * @author Dominik Brader <support@findologic.com>
+ */
+
+$lang['menu'] = 'FINDOLOGIC XML Export Plugin';
+$lang['youCan'] = 'Man kann den ';
+$lang['callExport'] = 'FINDOLOGIC XML Export manuell aufrufen';
+$lang['noTitleWarning'] = 'Diese Seiten scheinen keinen Titel zu haben.';
+$lang['noTitleWarningMoreInformation'] = 'Bitte stellen Sie sicher, dass jede Seite einen Titel gesetzt hat.';
+$lang['pagesWithoutTitle'] = 'Seiten ohne Titel';
+$lang['wikipage'] = 'Wikiseite';
+$lang['url'] = 'URL';
+$lang['lasteditby'] = 'Zuletzt editiert von';
+$lang['lastedited'] = 'Zuletzt editiert';
+$lang['edit'] = 'Editieren';
+$lang['thereAre'] = 'Es gibt ';
+$lang['morePages'] = ' weitere Seite(n), die keinen Titel gesetzt hat/haben.';
+$lang['allPagesHaveATitle'] = 'Alle Seiten haben einen Titel gesetzt! Es muss nichts angepasst werden.';
+$lang['noTitleTooltip'] = 'Sollten Seiten keinen Titel gesetzt haben, kann man diese in der Suche nur über die Kategorien finden. Außerdem haben Seiten die keinen Titel haben auch in der Suchresultatsseite keinen gesetzten Namen. ';
```

Comment:
Is this true? Are pages without title being exported at all?

Reply:
As far as I can tell we always said that they should get exported, but the user should be notified in the admin page.
In `\DokuwikiXMLExport::getPageIds` we only exclude pages that do not have a description:

```php
$pagesAndDeletedPages = array_filter($pagesAndDeletedPages, function ($page, $k) {
    return (p_get_metadata($page)['description'] !== '');
}, ARRAY_FILTER_USE_BOTH);
```

We could change the behavior, but is this how we want the behavior should be? 
Lets discuss this!

Reply:
As discussed I will implement it as you suggested.

Reply:
@georgms I will only be able to make the commit tomorrow, because I need to fix ~45% of all tests due to this change.

Reply:
:white_check_mark: